pkgs/desktops/gnome-3/extensions/pop-os-shell/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, nodePackages, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "pop-os-shell";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "shell";
+    rev =  "${version}";
+    sha256 = "sha256-gv9knhG1/HOUsIyPWfBRBpJWg/bfRXbbg/x2QDgSmls=";
+  };
+
+  buildInputs = [ glib nodePackages.typescript ];
```

these are likely just need for glibschemas and running ts
```suggestion
  nativeBuildInputs = [ glib nodePackages.typescript ];
```